src/rules/objectLiteralShorthandRule.ts

```diff
@@ -31,8 +31,8 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static LONGHAND_PROPERTY = "Expected property shorthand in object literal.";
-    public static LONGHAND_METHOD = "Expected method shorthand in object literal.";
+    public static LONGHAND_PROPERTY = "Expected property shorthand in object literal ('{foo, bar}').";
+    public static LONGHAND_METHOD = "Expected method shorthand in object literal ('{foo() {...}}').";
```

Comment:
I have a different PR for that, though only for the shorthand property - the functions are tricky to find in the AST.